scripts/test124.js

```diff
@@ -0,0 +1,21 @@
+const abi = require('../app/client/TokenPriceDataFeed.abi')
```

Comment:
did you want to keep this file? looks like it might have been a debugging thing

Reply:
ohh I didn't mean to push this